src/components/Header/Header.tsx

```diff
@@ -0,0 +1,13 @@
+import { HeaderContainer } from "./Header.styled";
+
+type HeaderProps = {
+  children: React.ReactNode;
+};
+
+export default function Header({ children }: HeaderProps) {
+  return (
+    <>
+      <HeaderContainer>{children}</HeaderContainer>
+    </>
+  );
```

Comment:
```suggestion
  return (
    <HeaderContainer>{children}</HeaderContainer>
  );
```

no need for using react fragments if you only return one child (only needed if you wanna return multiple top level children)

 Usually react fragments get rendered to a `<div>` in the output (and not stripped out!) so it's best to only use them if you need them to reduce the html elements used in the output

Reply:
✅